fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -50,11 +50,11 @@ def self.print_date
     end
 
     def self.print_loaded_plugins
-      ENV["FASTLANE_ENV_PRINTER"] = "enabled"
```

Comment:
Is this env var now deprecated / unused?

Reply:
This ENV var wasn't used by anything external at all 😬 Its only use being used as a parameter of sorts 😔